Assignment2/Cristinalonso29-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "prefix ": http://example.org/elements#,
+    "xsd": http://www.w3.org/2001/XMLSchema#

```

Comment:
prefix URIs must be defined between " "
Also, there is an extra blank space in "prefix "